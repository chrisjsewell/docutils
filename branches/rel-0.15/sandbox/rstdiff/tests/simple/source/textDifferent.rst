.. role:: change-replaced
.. role:: change-replacement

:change-replaced:`Some`:change-replacement:`A changed` line.
