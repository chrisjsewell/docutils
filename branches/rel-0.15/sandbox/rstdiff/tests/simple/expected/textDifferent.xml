<?xml version="1.0" encoding="utf-8"?>
<!-- Generated by Docutils 0.8 -->
<document source="input/textDifferent.old.rst =&gt; input/textDifferent.new.rst">
    <paragraph>
        <inline classes="change-replaced">
            Some
        </inline>
        <inline classes="change-replacement">
            A changed
        </inline>
         line.
    </paragraph>
</document>
